jetstream/README.md

```diff
@@ -603,6 +609,193 @@ ackF, err = js.PublishAsync("ORDERS.new", []byte("hello"))
 Just as for synchronous publish, `PublishAsync()` and `PublishMsgAsync()` accept
 options for setting headers.
 
+## KeyValue Store
+
+JetStream KeyValue Stores offer a straightforward method for storing key-value
+pairs within JetStream. These stores are supported by a specially configured
+stream, designed to efficiently and compactly store these pairs. This structure
+ensures rapid and convenient access to the data.
+
+The KV Store, also known as a bucket, enables the execution of various operations:
+
+- create/update a value for a given key
+- get a value for a given key
+- delete a value for a given key
+- purge all values from a bucket
+- list all keys in a bucket
+- watch for changes on given key set or the whole bucket
+- retrieve history of changes for a given key
+
+### Basic usage of KV bucket
+
+The most basic usage of KV bucket is to create or retrieve a bucket and perform
+basic CRUD operations on keys.
+
+```go
+js, _ := jetstream.New(nc)
+ctx := context.Background()
+
+// Create a new bucket. Bucket name is required and has to be unique within a stream.
```

Comment:
The bucket is a stream, so not sure what it means?

Reply:
Yeah, that does not make sense - it was meant to be "within a JetStream account".